server/migrations/20171119181507-create-event.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Events', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    name: {
+      type: Sequelize.STRING
+    },
+    venue: {
+      type: Sequelize.STRING
+    },
+    date: {
+      type: Sequelize.DATEONLY
+    },
+    time: {
+      type: Sequelize.STRING
+    },
+    createdAt: {
+      allowNull: false,
+      type: Sequelize.DATE
+    },
```

Comment:
please do not push migrations files

Reply:
Thank you for the feedback. I will remove it